website/pages/component/toast/guidelines.mdx

```diff
@@ -0,0 +1,229 @@
+        priority: 'Medium priority',
+        userAction:
+          'Optional: Banners remain until dismissed by the user, or if the state that caused the banner is resolved',
+      },
+      {
+        component: 'Modal',
+        priority: 'Highest priority',
+        userAction:
+          'Required: Modals block usage until the user takes a modal action or exits the modal (if available)',
+      },
+    ]}
+    columns={[
+      <TableHeader
+        key="component"
+        label="Component"
+        className={css`
+          width: 100px;
+        `}
+      />,
+      <TableHeader
+        key="priority"
+        label="Priority"
+        className={css`
+          width: 100px;
+        `}
+      />,
+      <TableHeader key="userAction" label="User Action" />,
+    ]}
+  >
+    {({ datum }) => (
+      <tr
+        key={datum.component}
+        className={css`
+          border-bottom: 1px solid ${uiColors.gray.light2};
+        `}
+      >
+        <Cell
+          className={css`
+            padding: 16px 0 16px 10px;
+          `}
+        >
+          {datum.component}
+        </Cell>
+        <Cell
+          className={css`
+            padding: 16px 0 16px 10px;
+          `}
+        >
+          {datum.priority}
+        </Cell>
+        <Cell
+          className={css`
+            padding: 16px 0 16px 10px;
+          `}
+        >
+          {datum.userAction}
+        </Cell>
+      </tr>
+    )}
+  </Table>
+</Card>
+
+Toasts can be used to confirm an action. For example, here’s a confirmation toast that shows up after an email has been sent.
+
+<div
+  className={css`
+    display: flex;
+    justify-content: flex-start;
+    flex-grow: 1;
+    width: 100%;
+  `}
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-1.png`}
+    alt=""
+    width="432px"
+    className={css`
+      margin-left: -16px;
+    `}
+  />
+</div>
+
+They can be used to indicate something destructive is or will be occurring.
+
+<div
+  className={css`
+    display: flex;
+    justify-content: flex-start;
+    flex-grow: 1;
+    width: 100%;
+  `}
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-2.png`}
+    alt=""
+    width="432px"
+    className={css`
+      margin-left: -16px;
+    `}
+  />
+</div>
+
+They can be used to indicate something important may need to be addressed.
+
+<div
+  className={css`
+    display: flex;
+    justify-content: flex-start;
+    flex-grow: 1;
+    width: 100%;
+  `}
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-3.png`}
+    alt=""
+    width="432px"
+    className={css`
+      margin-left: -16px;
+    `}
+  />
+</div>
+
+They can be used to show general information that may be helpful to the user.
+
+<div
+  className={css`
+    display: flex;
+    justify-content: flex-start;
+    flex-grow: 1;
+    width: 100%;
+  `}
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-4.png`}
+    alt=""
+    width="432px"
+    className={css`
+      margin-left: -16px;
+    `}
+  />
+</div>
```

Comment:
would other/future guidelines benefit from defining a component for this?

Reply:
I'm not sure yet to be honest. Maybe we leave for now and as I build out more guidelines I will abstract out if necessary.